docs/src/main/sphinx/functions/ai.md

```diff
@@ -57,7 +100,11 @@ ai.model=claude-3-5-sonnet-latest
 ai.anthropic.api-key=xxx
 ```
 
-### OpenAI
+Use [secrets](/security/secrets) to avoid actual API KEY values in the catalog
```

Comment:
Why upper case for KEY? Do we do that elsewhere?

Reply:
typo - fixed